### ISSUE TITLE
frame-focusable-content should ignore negative tabindex

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#3466]
The `frame-focusable-content` rule should not fail [Passed Example 2](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/akn7bn/aa153f6799d28563054ce66bcf7dfcedf9b75288.html) of the [iframe with negative tabindex has no interactive elements]() ACT rule.

```html
<iframe tabindex="-1" srcdoc="
  <a href='/' tabindex='-1'>Home</a>
"></iframe>
```

**Comment by padmavemulapati** [comment]
Not seeing any findings of `frame-focusible-content` except in `inapplicable`
<img width="1225" alt="image" src="https://user-images.githubusercontent.com/47409579/187349834-1a5f378b-79a1-41a8-95a6-6a9164e273b1.png">

**Comment by straker** [comment]
@padmavemulapati what does your test HTML look like?

**Comment by padmavemulapati** [comment]
Validated with the latest develop branch code base, when we run axe inside iframes and frame has its focussible content , then 
`frame-focussible-content` rule is failing instead passing
